docs/requirements.md

```diff
@@ -95,8 +85,18 @@ You must manually create the databases required by Codacy on the PostgreSQL serv
 
     ```sql
     CREATE USER codacy WITH PASSWORD 'codacy';
-    ALTER ROLE codacy WITH CREATEDB;
+    ALTER ROLE codacy WITH CREATEDB
+    ```
+
+3.  Make sure that you can connect to the PostgreSQL database using the newly created user. For example:
+
+    ```bash
+    psql -U codacy -h <PostgreSQL server hostname>
```

Comment:
I think this command will fail if there is no database with the same name as the user.